examples/todoapp/lib/lib_translations/data_sources/local/translations_local_data_source.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:convert';
+
+import '../../../app_extensions.dart';
+import '../../../base/data_sources/local/shared_preferences_instance.dart';
+import '../../models/i18n_models.dart';
+import '../translations_data_source.dart';
+
+class TranslationsLocalDataSource implements TranslationsDataSource {
+  TranslationsLocalDataSource(this.sharedPreferences);
+
+  final SharedPreferencesInstance sharedPreferences;
+
+  Future<bool> saveTranslations(I18nSections? translations) async {
+    if (translations == null) {
+      return sharedPreferences.setString(translationsKey, '');
```

Comment:
As discussed the translations are already provided from the arb files so there is no need of additional caching

Reply:
Right, but currently at each app reset the request to the translations API is executed, which when fails displays the error.

Reply:
Let's ignore that error. The intent of this remote data is to override some of the texts within the app, so regardless of the status of the API (working or not) the application should working just fine

Reply:
Done